Pod/Native/NativeScenario.swift

```diff
@@ -11,29 +11,43 @@ import Foundation
 class NativeScenario : CustomStringConvertible {
     let scenarioDescription: String
     let stepDescriptions: [String]
+    let index: Int
 
     /**
      If the scenario description is 'Test funny things are funny' then the result of calling
      `selectorName` would be `testTestFunnyThingsAreFunny`
      */
     var selectorString: String {
-        get { return "test\(self.scenarioDescription.camelCaseify)" }
+        get { return "test\(self.leftPad(index))\(self.scenarioDescription.camelCaseify)" }
     }
     
     var selectorCString: UnsafeMutablePointer<Int8> {
         get { return strdup(self.selectorString) }
     }
     
-    required init(_ description: String, steps: [String]) {
+    required init(_ description: String, steps: [String], index: Int = 0) {
         self.scenarioDescription = description
         self.stepDescriptions = steps
+        self.index = index
     }
     
     var description: String {
         get {
             return "<\(type(of: self)) \(self.selectorString) \(self.stepDescriptions.count) steps>"
         }
     }
+    
+    private func leftPad(_ index: Int) -> String {
+        guard index > 9 else {
```

Comment:
They're not riveting reads, but this https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/Strings/Articles/formatSpecifiers.html and this http://pubs.opengroup.org/onlinepubs/009695399/functions/printf.html are handy resources to know about. Though I have to look them up every time I need something other than `%f` :)